### PR TITLE
Add PGL to CPS whitelist

### LINKED ISSUE #2559: Add PGL to CPS type whitelist
~~Blocked on https://github.com/bbc/psammead/issues/1232~~
~~Blocked on https://github.com/bbc/simorgh/issues/2561 (transitively externally blocked)~~
Blocked waiting for UX review

**Is your feature request related to a problem? Please describe.**
Since we're adding support for photo galleries on our front pages, we now need to stop filtering out PGLs.

**Describe the solution you'd like**
Add `PGL` to the whitelist array in [cpstypes.js](https://github.com/bbc/simorgh/blob/latest/src/app/lib/utilities/preprocessor/rules/cpstypes.js) and enable the use of photo gallery media indicator on in story promo

**Describe alternatives you've considered**
📷🚫 = 👎 

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**
WSFRONTPGS-83

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #2559

**Overall change:**
Add `PGL` to the CPS whitelist and enable the use of the photo gallery media indicator in the Story Promos.

**Code changes:**
- Add `PGL` to the whitelist array
- Apply `photogallery` type in the `MediaIndicator` if `isPGL`
- Bump `StoryPromo` and `MediaIndicator` version
- Add and update tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — APPROVED** [review]

**Review by dr3 — CHANGES_REQUESTED** [review]
LGTM, but pulls in multiple versions of @bbc/psammead-styles

**Review by j-pendlebury — APPROVED** [review]

**Comment by andrew-nowak** [comment]
Still has the psammead-styles duplication... but it looks like it should get resolved with an `npm dedupe`?

File: src/app/containers/StoryPromo/MediaIndicator/index.test.jsx [hunk 1]
```diff
@@ -21,6 +39,16 @@ const item = {
...
 };
 
+const photogalleryItem = {
+  headlines: {
+    headline: 'A photo gallery item',
+  },
+  locators: {
+    assetUri: 'https://www.bbc.co.uk',
+  },
+  cpsType: 'PGL',
```
Comment (ghost) [inline src/app/containers/StoryPromo/MediaIndicator/index.test.jsx:49]: Why are we not including `media.format` for `PGL`?

Comment (DenisHdz) [reply src/app/containers/StoryPromo/MediaIndicator/index.test.jsx:49]: The `PGL` cpsType doesn't have a media object in the feed. 
https://github.com/bbc/simorgh/blob/latest/data/pidgin/frontpage/index.json#L4868

**Review by ghost — APPROVED** [review]
LGTM. 👍

**Review by dr3 — APPROVED** [review]
LGTM other than conflict

**Comment by david-boydell** [comment]
Cool, good stuff this rfm.

**Pull request merged**